emain/emain-window.ts

```diff
@@ -73,11 +73,32 @@ export class WaveBrowserWindow extends BaseWindow {
     private actionQueue: WindowActionQueueEntry[];
 
     constructor(waveWindow: WaveWindow, fullConfig: FullConfigType, opts: WindowOpts) {
+        const settings = fullConfig?.settings;
+
         console.log("create win", waveWindow.oid);
         let winWidth = waveWindow?.winsize?.width;
         let winHeight = waveWindow?.winsize?.height;
         let winPosX = waveWindow.pos.x;
         let winPosY = waveWindow.pos.y;
+
+        if (
+            (winWidth == null || winWidth === 0 || winHeight == null || winHeight === 0) &&
+            settings?.["window:dimensions"]
+        ) {
+            const dimensions = settings["window:dimensions"];
+            const [dimensionWidth, dimensionHeight] = dimensions.split(/[xX]/);
+
+            const parsedWidth = parseInt(dimensionWidth);
+            const parsedHeight = parseInt(dimensionHeight);
+
+            if ((!winWidth || winWidth === 0) && !isNaN(parsedWidth)) {
+                winWidth = parsedWidth;
+            }
+            if ((!winHeight || winHeight === 0) && !isNaN(parsedHeight)) {
+                winHeight = parsedHeight;
+            }
+        }
+
         if (winWidth == null || winWidth == 0) {
```

Comment:
I see you haven't added the zoom factor yet, do you want to incorporate that into this PR or a follow-up? Could you also add the new settings to `config.mdx`?

Reply:
I will incorporate the zoom factor in a follow-up PR.